src/components/atoms/Input/Input.styled.ts

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+import colors from 'styles/colors';
+import typography from 'styles/typography';
+
+export const InputWrapper = styled.div`
+  box-sizing: border-box;
```

Comment:
box-sizing should be available for whole page. I think we can move it to global styles